ts/sagas/contentLoaders.ts

```diff
@@ -18,11 +19,16 @@ const contentClient = ContentClient();
  */
 function getServiceMetadata(
   serviceId: ServiceId
-): Promise<t.Validation<BasicResponseType<ServiceMetadata>>> {
+): Promise<Either<Error, t.Validation<BasicResponseType<ServiceMetadata>>>> {
```

Comment:
I'm pretty sure this new type could be could quite confusing: you will have a left case with an error (could be ok) and a right case with another either with its left and its right.
You can leave type as it is`t.Validation<BasicResponseType<ServiceMetadata>>` and in reject case return a left of [Errors](https://github.com/gcanti/io-ts/blob/4d13500147bfd654c3db2d6f18a02334fbae44db/src/index.ts#L37)

Reply:
Fix in 6587733 thanks for support